From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: musa <musa@local>
Date: Thu, 4 Jun 2026 02:46:36 +0000
Subject: [PATCH] MUSA(csrc): csrc/quantization/w8a8/fp8/common.cuh

---
 csrc/quantization/w8a8/fp8/common.cuh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/csrc/quantization/w8a8/fp8/common.cuh b/csrc/quantization/w8a8/fp8/common.cuh
index 087f50991..e8eb04289 100644
--- a/csrc/quantization/w8a8/fp8/common.cuh
+++ b/csrc/quantization/w8a8/fp8/common.cuh
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "libtorch_stable/quantization/vectorization.cuh"
-#include "../../utils.cuh"
+#include "quantization/utils.cuh"
 
 #include <cmath>
 
