From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: musa <musa@local>
Date: Thu, 4 Jun 2026 02:46:37 +0000
Subject: [PATCH] MUSA(csrc): csrc/attention/attention_kernels.cuh

---
 csrc/attention/attention_kernels.cuh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/csrc/attention/attention_kernels.cuh b/csrc/attention/attention_kernels.cuh
index 052ff168c..9da92379a 100644
--- a/csrc/attention/attention_kernels.cuh
+++ b/csrc/attention/attention_kernels.cuh
@@ -22,8 +22,8 @@
 #include <c10/cuda/CUDAGuard.h>
 #include <algorithm>
 
-#include "attention_dtypes.h"
-#include "attention_utils.cuh"
+#include "attention_musa/attention_dtypes.h"
+#include "attention_musa/attention_utils.cuh"
 #include "../cuda_compat.h"
 
 #ifdef USE_ROCM
@@ -31,7 +31,7 @@
   #include "../quantization/w8a8/fp8/amd/quant_utils.cuh"
 typedef __hip_bfloat16 __nv_bfloat16;
 #else
-  #include "../quantization/w8a8/fp8/nvidia/quant_utils.cuh"
+  #include "../quantization_musa/w8a8/fp8/nvidia/quant_utils.cuh"
 #endif
 
 #define MAX(a, b) ((a) > (b) ? (a) : (b))
